nf_core/gitpod/gitpod.Dockerfile

```diff
@@ -47,23 +46,20 @@ RUN conda config --add channels defaults && \
     conda config --add channels conda-forge && \
     conda config --set channel_priority strict && \
     conda install --quiet --yes --name base \
-    mamba \
```

Comment:
I wonder if someone uses mamba to run tests?

Reply:
we will find out 🙂